protobuf-c/protobuf-c.c

```diff
@@ -2629,16 +2629,16 @@ parse_oneof_member (ScannedMember *scanned_member,
 
 	/* If we have already parsed a member of this oneof, free it. */
 	if (*oneof_case != 0) {
+		const ProtobufCFieldDescriptor *old_field; size_t el_size;
```

Comment:
Please put the `el_size` declaration on a separate line

Reply:
Done!